### DISCUSSION TITLE
Is there any real reason to block PR #69067 forever ?

### DISCUSSION THREAD

**Discussion by pedrolcl** [microsoft/winget-pkgs#71565] [Q&A] [answered]
I've opened this PR #69067 in Aug 8 2022 using `winget-create' which generated a Manifest-Validation-Error, because the manifest was using the 1.2.0 schema. That was solved when that schema was enabled. The PR was approved by a moderator ( @ItzLevvie, who also approved the old version 1.6.0 ). But now, the error is Validation-Unapproved-URL. Why? Because it is hosted by SourceForge?  Do I need to close the PR and forget the matter?

**Comment by vedantmgoyal9** [comment, marked as answer]
The PR has been merged now.
